src/pages/_document.tsx

```diff
@@ -1,9 +1,11 @@
 import Document, { Head, Html, Main, NextScript } from 'next/document'
+import { getDarkMode } from 'src/styles/mediaQueries'
 
 class MyDocument extends Document {
   render() {
+    const isDarkMode = getDarkMode()
```

Comment:
Should this be `useDarkMode`? Otherwise it makes those `window` calls on every re-render